server/src/test/java/org/elasticsearch/search/aggregations/metrics/AvgAggregatorTests.java

```diff
@@ -611,14 +611,8 @@ public void testCacheAggregation() throws IOException {
         }
```

Comment:
Can you revert this change ? It should work without this modification so I'd like to keep this for a different pr since the issue is not related to the exitable directory reader.

Reply:
Sure, will do.